### PR TITLE
landing: improve responsivess and copy

### PR DESCRIPTION
**Pull request by Diogomartf**
- better display text on mobile
- add missing pontuaction
- Try Stackly banner
  - fix image overlapping button

### CODE REVIEW

File: packages/landing/app/page.tsx [hunk 1]
```diff
@@ -76,7 +77,7 @@ export default function Home() {
           <div className="max-w-xl px-6 space-y-2 md:pb-28">
             <HeadingText size={4}>Using Stackly is super easy</HeadingText>
             <HeadingText weight="regular" className="text-em-med">
-              Create a stack (aka. recurring buy) in 3 steps.
+              Create a stack (aka. recurring buys) in 3 steps.
```
Comment (ElRodrigote) [inline packages/landing/app/page.tsx:80]:
```suggestion
              Create a stack (aka. recurring swaps) in 3 steps.
```

File: packages/landing/app/page.tsx [hunk 2]
```diff
@@ -146,7 +147,7 @@ export default function Home() {
                 regardless of the {"asset's"} price.
               </QAndAAccordion>
               <QAndAAccordion question="Why one should do DCA?">
-                Recurring buys (aka DCA) removes the need to time the market,
+                Recurring buys (aka. DCA) removes the need to time the market,
```
Comment (ElRodrigote) [inline packages/landing/app/page.tsx:150]:
```suggestion
                Recurring swaps (aka DCA) remove the need to time the market,
```

File: packages/landing/app/page.tsx (hunk 1 above)
Comment (berteotti) [reply packages/landing/app/page.tsx:80]: Also "aka" doesn't have a dot at the end.

Create a stack (aka recurring buys/swaps) in 3 steps.

Comment (ElRodrigote) [reply packages/landing/app/page.tsx:80]: All of the following are accepted AFAIK:
* `A.K.A`
* `AKA.`
* `AKA`
* `a.k.a`
* `aka.`
* `aka`

Comment (berteotti) [reply packages/landing/app/page.tsx]: Quick search shown me `aka` is the one. But I don't mind, anyone is fine if users understand.

**Review by ElRodrigote — DISMISSED** [review]

**Pull request merged**